style/cell.less

```diff
@@ -4,8 +4,8 @@
   position: absolute;
   height: inherit;
   padding: 0 8px;
-  border-right: 1px solid @borderColor;
-  border-bottom: 1px solid @borderColor;
+  border-right: 1px solid var(--border-color);
```

Comment:
I wonder if we should use `--rdg` prefix? I am fine with either

Reply:
It is scoped under `.rdg`, so it's probably fine I think.